src/local-history/local-history-manager.ts

```diff
@@ -139,38 +143,40 @@ export class LocalHistoryManager {
      * Save the current context of the active editor.
      */
     public async saveEditorContext(document: vscode.TextDocument): Promise<void> {
-        const timestamp = this.getCurrentTime();
-        const timestampForFileName = timestamp.replace(/[-:. ]/g, '');
-        const fileFullPath = path.parse(document.fileName);
-        const historyFileName = `${fileFullPath.name}_${timestampForFileName.substring(0, 14)}_${timestampForFileName.substring(14, 17)}${fileFullPath.ext}`;
-        const hashedFolderPath = this.getHashedFolderPath(document.fileName);
-
-        // Create a folder (and all the parent folders) for storing all the local history file for the active editor.
-        if (!fs.existsSync(hashedFolderPath)) {
-            fs.mkdirSync(hashedFolderPath, { recursive: true });
-        }
+        if (this.fileSizeLimit(document)) {
```

Comment:
Should we at least log when we attempt to save a file which is too large?

Reply:
wdym? we plan not to save when the file is too large out of our file size limit range.